upb/def.c

```diff
@@ -2418,6 +2468,18 @@ static void create_enumdef(
   }
```

Comment:
this function is getting a bit long.  Can you pull the contents of this loop into a well named helper?  (follow on change is fine)